cmd/machine-config-operator/bootstrap.go

```diff
@@ -24,24 +24,25 @@ var (
 	}
 
 	bootstrapOpts struct {
+		cloudConfigFile      string
```

Comment:
this is just adding cloudConfigFile and alphabetizing bootstrapOpts?

Reply:
Yes